### PR TITLE
Add metadata to Docker images

### PR DESCRIPTION
**Pull request by e0ne**
For troubleshooting it's useful to have following data:
* image build time
* git branch
* git revision

Also this commits modifies 'make image' target to build webhook image too.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -69,8 +76,21 @@ update-codegen:
...
+	$(IMAGE_BUILDER) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+		--build-arg VERSION="$(BUILD_VERSION)" \
+		--build-arg VCS_REF="$(VCS_REF)" \
+		--build-arg VCS_BRANCH="$(VCS_BRANCH)" \
+		-f $(DOCKERFILE_CONFIG_DAEMON) -t $(CONFIG_DAEMON_IMAGE_TAG) $(CURPATH) $(IMAGE_BUILD_OPTS)
+	$(IMAGE_BUILDER) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+		--build-arg VERSION="$(BUILD_VERSION)" \
+		--build-arg VCS_REF="$(VCS_REF)" \
+		--build-arg VCS_BRANCH="$(VCS_BRANCH)" \
+		-f $(DOCKERFILE_WEBHOOK) -t $(WEBHOOK_IMAGE_TAG) $(CURPATH) $(IMAGE_BUILD_OPTS)
```
Comment (bn222) [inline Makefile:93]: Can we add the vars inside the IMAGE_BUILDER var?

Comment (e0ne) [reply Makefile:93]: done

**Comment by bn222** [comment]
LGTM

File: Makefile [hunk 2]
```diff
@@ -12,8 +12,15 @@ export OPERATOR_EXEC?=oc
 BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 IMAGE_BUILDER?=docker
 IMAGE_BUILD_OPTS?=
+IMAGE_METADATA=--build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" --build-arg VERSION="$(BUILD_VERSION)" --build-arg VCS_REF="$(VCS_REF)" --build-arg VCS_BRANCH="$(VCS_BRANCH)"
```
[the remark this answers is not in the document]
Comment (zeeke) [reply Makefile:15]: In that sense, I suggest to update how images are built for end-to-end tests:

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/hack/run-e2e-conformance-virtual-cluster.sh#L265 

This way we can see it in CI

[the remark this answers is not in the document]
Comment (SchSeba) [reply Makefile:15]: +1 lets do the same build as part of the CI

**Comment by adrianchiris** [comment]
@e0ne you still planning to work on this one ?

**Comment by e0ne** [comment]
We don't need this PR anymore

**Pull request closed without merging**